install_pim/manual/system_requirements/system_requirements.rst.inc

```diff
@@ -31,33 +31,31 @@ The web server will also need the following libraries and modules:
 **PHP required modules and configuration**
 
 +----------------+---------------------------+
-| PHP            | 7.1                       |
+| PHP            | 7.2                       |
 +----------------+---------------------------+
 | php-apcu       | apc.enable_cli=1          |
 +----------------+---------------------------+
-| php7.1-bcmath  | No specific configuration |
+| php7.2-bcmath  | No specific configuration |
 +----------------+---------------------------+
-| php7.1-curl    | No specific configuration |
+| php7.2-curl    | No specific configuration |
 +----------------+---------------------------+
-| php7.1-fpm     | No specific configuration |
+| php7.2-fpm     | No specific configuration |
 +----------------+---------------------------+
-| php7.1-gd      | No specific configuration |
+| php7.2-gd      | No specific configuration |
 +----------------+---------------------------+
-| php7.1-intl    | No specific configuration |
+| php7.2-intl    | No specific configuration |
 +----------------+---------------------------+
-| php7.1-mcrypt  | No specific configuration |
+| php7.2-mysql   | No specific configuration |
```

Comment:
is not mysql module listed just after? is mcrypt no longer necessary?

Reply:
`mcrypt` is no longer necessary, and does not exists anymore in PHP 7.2 anyway.
For MySQL, indeed, it is noted twice, the second one is in the "MySQL" part, the first in the "PHP" part. It is in case the dev looks just for a part of the requirements.